### PR TITLE
update to new VS 2017 csproj format

### PR DESCRIPTION
**Pull request by SimonCropp**
this is in prep for targeting net standard.

### CODE REVIEW

**Comment by SimonCropp** [comment]
@Particular/azure-maintainers this is ready to go

File: GitVersion.yml [hunk 1]
```diff
@@ -3,5 +3,3 @@ next-version: 7.0
 branches:
   develop:
     tag: alpha
-  releases?[/-]:
```
Comment (danielmarbach) [inline GitVersion.yml]: Is this no longer supported @SimonCropp ?

Comment (SimonCropp) [inline GitVersion.yml]: yeah the regex approach seems to have been dropped from gitversion. got some errors

**Comment by danielmarbach** [comment]
@SimonCropp thanks!

**Pull request merged**